src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,81 @@
+import * as SearchClient from "../clients/SearchClient";
+import * as URLHelper from "../helpers/URLHelper";
+
+function updateTable(tableStore) {
+  return function (tableStateObject) {
+    tableStore.setRows(tableStateObject.results);
+    tableStore.setCurrentPage(tableStateObject.current_page);
+    tableStore.setTotalPages(tableStateObject.total_pages);
+    tableStore.emitChange();
+  };
+}
+
+export class FilterBarActor {
+  constructor(filterBarStore, tableStore) {
+    this.filterBarStore = filterBarStore;
+    this.tableStore = tableStore;
+  }
+
+  enableFilter(filterUid, value) {
+    this.filterBarStore.enableFilter(filterUid, value);
+  }
+
+  disableFilter(filterUid) {
+    this.filterBarStore.disableFilter(filterUid);
+  }
+
+  disableAllFilters() {
+    this.filterBarStore.disableAllFilters();
+    this.applyFilters();
+  }
+
+  updateFilter(filterUid, propKey, propValue) {
+    this.filterBarStore.updateFilter(filterUid, propKey, propValue);
+  }
+
+  applyFilters() {
+    var url = URLHelper.updateUrlSearch(
+                this.filterBarStore.getSearchUrl(),
+                "q",
+                this.filterBarStore.getQuery()
+              ).toString();
+
+    this.tableStore.setUrl(url);
```

Comment:
Might have to be for 2.0. I know it carries ambiguity, but it's a side effect of not having a top level URL store and actor.